projects/storefrontapp-e2e-cypress/cypress/integration/regression/my-account/close-account.e2e-spec.ts

```diff
@@ -2,28 +2,36 @@ import {
   closeAccountTest,
   verifyAsAnonymous,
 } from '../../../helpers/close-account';
-import { formats } from '../../../sample-data/viewports';
 
-describe('Close Account page', () => {
+describe('My Account - Close Account', () => {
   before(() =>
     cy.window().then(win => {
       win.sessionStorage.clear();
     })
   );
 
-  verifyAsAnonymous();
-  closeAccountTest();
-});
-
-describe(`${formats.mobile.width + 1}p resolution - Close Account page`, () => {
-  before(() => {
-    cy.window().then(win => win.sessionStorage.clear());
-    cy.viewport(formats.mobile.width, formats.mobile.height);
-  });
-  beforeEach(() => {
-    cy.viewport(formats.mobile.width, formats.mobile.height);
+  describe('close account test for anonymous user', () => {
+    verifyAsAnonymous();
   });
 
-  verifyAsAnonymous();
-  closeAccountTest();
+  describe('close account test for logged in user', () => {
+    before(() => {
+      cy.requireLoggedIn();
```

Comment:
we need to remove the registerAndLogin from the close-account.ts because this already does it

Reply:
Maybe the others as well, and I believe it should be in the before to keep the test consistent

Reply:
the test doesn't work if we use requireLoggedIn(StandardUser) here. 

The only tests that need the standard user are the close account, update email and update password.

Reply:
Sorry I didn't explain it properly. In each of those files you mentioned, close account, update email, and update password has inside their myaccountTest() function a function called registerAndLogin(); 

- This is doing the same as above which makes it login twice, therefore we need to remove the top one for those mentioned test case.
- Also, to make it with with the standard user for the code above, it is better to remove the visit '/' inside the registerAndLogin() function, and replace the code above with that function so the need for standard user will still be used and it will no longer do two logins. So, instead of cy.requireLoggedIn in those mentioned case, you will have to import registerAndLogin and use that instead of the one above.